conan/cli/commands/list.py

```diff
@@ -240,6 +241,8 @@ def list(conan_api: ConanAPI, parser, *args):
             raise ConanException("Cannot define lru when loading a graph json file")
         if args.filter_profile or args.filter_settings or args.filter_options:
             raise ConanException("Filtering binaries cannot be done when loading a graph json file")
+        if not os.path.isfile(args.graph):
```

Comment:
This is not as much a cli arg error as the others.
Maybe it would be better to make ``MultiPackageLists.load_graph(graph_file`` be the one that raises the error if ``graphfile`` doesn't exist? That will make a more robust API too.

Reply:
Totally right